helm/cas-metabase/values.yaml

```diff
@@ -72,3 +72,11 @@ networkPolicies:
 
 restoreFromProd:
   enable: false
+
+download-cas-metabase-dags:
+  image:
+    tag: 69f346b134d99e9e5e6d50e16198fd4977fc3264 # pragma: allowlist secret
```

Comment:
do we need to override the default value for this image?

Reply:
I think now that the chart is released properly we won't have to